homeassistant/components/tractive/__init__.py

```diff
@@ -56,7 +63,17 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     tractive = TractiveClient(hass, client, creds["user_id"])
     tractive.subscribe()
 
-    hass.data[DOMAIN][entry.entry_id] = tractive
+    trackable_objects = await client.trackable_objects()
+    try:
+        trackables = await asyncio.gather(
```

Comment:
`asyncio.gather` raises on an empty list.

Reply:
I think `asyncio.gather` returns an empty list when list of task is empty. Or maybe I don't understand your comment.

Reply:
Try it 🙃 await asyncio.gather() raises

Reply:
I'm confised because I tried 🤔 This code doesn't raises.
```python
import asyncio

async def main():
    tasks = []
    result = await asyncio.gather(*tasks)
    print(result)

    result2 = await asyncio.gather()
    print(result2)

loop = asyncio.get_event_loop()
loop.run_until_complete(main())
loop.close()
```

Reply:
@balloob Can we proceed? I think PR is ready to merge.